tox.ini

```diff
@@ -1,47 +1,53 @@
 [tox]
 envlist = py27,pypy,py33,py34,py35,flake8
+deps =
+    pytest
+    unicodecsv
+    hypothesis>=3.6.1,<4.0.0
+
 [testenv:py27]
 whitelist_externals = rm
-commands = 
+commands =
     rm -f jellyfish/cjellyfish.so
     pip install -e .
-    py.test jellyfish/test.py
-deps = pytest
-    unicodecsv
+    pytest jellyfish/test.py jellyfish/test_properties.py
```

Comment:
pytest updated its invocation method to drop the confusing `.` so I've updated this accordingly.